openmc/data/multipole.py

```diff
@@ -112,7 +112,8 @@ def _broaden_wmp_polynomials(E, dopp, n):
         erf_beta = 1.0
         exp_m_beta2 = 0.0
     else:
-        erf_beta = np.erf(beta)
+        from scipy.special import erf
```

Comment:
Yes, we should be using `math.erf`. Can you make that change and also use `math.sqrt`, `math.exp`, and `math.pi` in this function? i.e., just have `from math import exp, erf, pi` at the top of the module. The numpy versions of those functions should only be used if the argument is a vector (in this case, it's all scalars)